pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -137,7 +128,7 @@ func getClients() (*vCenterClient, error) {
 
 	// There is a noticeable delay when creating the client, so let the user know what's going on.
 	logrus.Infof("Connecting to vCenter %s", vcenter)
-	vim25Client, restClient, err := vspheretypes.CreateVSphereClients(context.TODO(),
+	vim25Client, restClient, err := CreateVSphereClients(context.TODO(),
```

Comment:
Other cloud platforms store the client in a session object in the install config (check out aws or azure). Would that approach work for vsphere?

Reply:
So technically, the installer is leaking session here (they never get logged out). We did some research on this, across OpenShift, in https://issues.redhat.com/browse/SPLAT-319 and actually meeting with the Govmomi community this week to see if improving session management _inside_ govmomi would be suitable. 

I'm a little hesitant to add yet another session management layer to an OpenShift component with the other work about to start that I mentioned above. 

If the govmomi community doesn't want to move forward with session management improvements in govmomi, I am certainly happy to revisit what you point out.

Reply:
> I'm a little hesitant to add yet another session management layer to an OpenShift component with the other work about to start that I mentioned above.

After meeting with the Govmomi community today, there won't be any work on session management improvements in the near future, due to lots of design work needed. I will plan on a separate PR to address session issues here if that's okay. I've created [SPLAT-471](https://issues.redhat.com/browse/SPLAT-471) for this task.